mapcomposer/app/static/externals/gxp/src/script/widgets/AdvancedScaleOverlay.js

```diff
@@ -176,8 +176,10 @@ gxp.AdvancedScaleOverlay = Ext.extend(Ext.Panel, {
             }else{ 
                 Ext.get("id_box").insertBefore(Ext.get("zoom_selector"));               
             }
-            this.getEl().on("click", this.stopMouseEvents, this);
-            this.getEl().on("mousedown", this.stopMouseEvents, this);
+			if(!this._fractionalZoom){
```

Comment:
I don't stop the events because, in this case, I can not edit the scale selector.
Do you know another solution?